src/icons/fileIcons.ts

```diff
@@ -2416,6 +2416,7 @@ export const fileIcons: FileIcons = {
       ],
     },
     { name: 'ifanr-cloud', fileNames: ['.mincloudrc'] },
+    { name: 'concourse', fileNames: ['pipeline.yml'], light: true },
```

Comment:
Unfortunately, we can't apply such a specific icon to files called 'pipeline.yml' as there might be some many projects outside which have the same files but no relation to concourse. Hopefully you understand this decision. Maybe you can still add this icon locally, check out this config: https://github.com/PKief/vscode-material-icon-theme?tab=readme-ov-file#custom-svg-icons

Reply:
are there any more specific file extensions of file names for this?

Reply:
[Concourse CI - Code Samples | Microsoft Learn](https://learn.microsoft.com/en-us/samples/azure/azure-quickstart-templates/concourse-ci/)

Is the `concourse.yml` mentioned here a convention, @LukeTimeWalker? It doesn't agree with [documentation](https://concourse-ci.org/setting-pipelines.html)

Reply:
Not really. I have only seen the name `pipeline.yml` in use so far. But we can use `concourse.yml` as this should be project specific enough.

Reply:
Yep, unfortunately `pipeline.yml` is popular with Buildkite too.

Reply:
@LukeTimeWalker I've another idea, there's an extension available which also provides some language IDs for concourse:

![image](https://github.com/PKief/vscode-material-icon-theme/assets/12248527/24fc8260-45fb-4592-a33e-526862a3adfb)

If you add a new definition in the language icons file, it should also work for "pipeline.yml" files:
https://github.com/PKief/vscode-material-icon-theme/blob/main/src/icons/languageIcons.ts

just add the following line:

```ts
{ icon: { name: 'concourse' }, ids: ['concourse-pipeline-yaml', 'concourse-task-yaml'] },
```

Reply:
just tried it out locally and it worked. VS Code also assigns icons by language ID:

![image](https://github.com/PKief/vscode-material-icon-theme/assets/12248527/733921e8-7961-449a-bd3a-738c47298407)

Reply:
Nice. Never heard of language IDs before. Is it possible to assign the icon by language ID and (logical OR) name (`concourse.yml`)?